packages/webapp/src/components/Drawer/index.tsx

```diff
@@ -46,23 +51,19 @@ const Drawer = ({
   buttonGroup,
   classes = {
     modal: '',
-    drawer: '',
     drawerBackdrop: '',
     drawerHeader: '',
     drawerContent: '',
     drawerContainer: '',
   },
   fullHeight,
-  responsiveModal = true,
+  desktopVariant = DesktopDrawerVariants.MODAL,
+  addBackdrop = true,
 }: DrawerProps) => {
   const theme = useTheme();
   const isDesktop = useMediaQuery(theme.breakpoints.up('sm'));
 
-  if (!isOpen) {
-    return null;
-  }
-
-  return isDesktop && responsiveModal ? (
+  return isDesktop && desktopVariant === DesktopDrawerVariants.MODAL && isOpen ? (
```

Comment:
I actually destroyed the mobile drawer slide-up animation with this change, which we did discuss a little in this [PR comment thread](https://github.com/LiteFarmOrg/LiteFarm/pull/3150#discussion_r1521902923) but kind of landed on thinking it was innocuous! At the time I totally missed that applying conditional rendering to the Drawer = no animation! Luckily it never went to App, but you can see the slide up animation broken on beta/integration now!

Reply:
Oooh great catch!!